pkg/registries/docker/metadata_v2.go

```diff
@@ -19,7 +19,7 @@ func HandleV2ManifestList(r *Registry, remote, ref string) (*storage.ImageMetada
 		// Default to linux arch
 		// TODO(ROX-13284): Support multi-arch images.
 		if manifest.Platform.OS == "linux" && manifest.Platform.Architecture == "amd64" {
-			return HandleV2Manifest(r, remote, manifest.Digest.String())
+			return handleManifests(r, manifest.MediaType, remote, manifest.Digest.String())
```

Comment:
Should we change the `HandleV2Manifest` call above this?

Reply:
Why would we?

Reply:
Is it not possible that the manifest list with only one item contains an OCI manifest? That being said, is it really even possible for a manifest list to only have one item...?

Reply:
Done, good point